deepkit-graphql.js.org

marcus-sa opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've just changed the subdomain inJS.ORGs zonefile. You may have to configure your host to accept the custom domain. As I got a error page from netlify I wonder why you are not using their default `<projectname>.netlify.app` as the target.
I'll merge after we have "contact". Please keep this PR open.

marcus-sa commented:
> I've just changed the subdomain inJS.ORGs zonefile. You may have to configure your host to accept the custom domain. As I got a error page from netlify I wonder why you are not using their default `<projectname>.netlify.app` as the target. I'll merge after we have "contact". Please keep this PR open.

I'm using https://mintlify.com not https://netlify.app (at least not directly).

I've added `deepkit-graphql.js.org` as a custom domain.

![Screenshot 2024-02-06 at 17 07 22](https://github.com/js-org/js.org/assets/8391194/7bad7fc4-57cc-4198-b26f-5699772944f1)

indus commented:
I don't thing we had a request for a mintlify hosted page before...
From what I can read here: https://mintlify.com/docs/settings/custom-domain/subdomain#configuring-your-dns
it looks like `cname.vercel-dns.com` is the target. So your request maybe has to look like this (?):
`"deepkit-graphql": "cname.vercel-dns.com"  // noCF`
Can you change your request so I can update JS.ORGs zonefile?

marcus-sa commented:
> I don't thing we had a request for a mintlify hosted page before... From what I can read here: https://mintlify.com/docs/settings/custom-domain/subdomain#configuring-your-dns it looks like `cname.vercel-dns.com` is the target. So your request maybe has to look like this (?): `"deepkit-graphql": "cname.vercel-dns.com" // noCF` Can you change your request so I can update JS.ORGs zonefile?

Done 👍🏼

indus commented:
Welcome back https://deepkit-graphql.js.org
